etcdserver/api/v3rpc/maintenance.go

```diff
@@ -128,11 +128,11 @@ func (ms *maintenanceServer) Snapshot(sr *pb.SnapshotRequest, srv pb.Maintenance
 }
 
 func (ms *maintenanceServer) Hash(ctx context.Context, r *pb.HashRequest) (*pb.HashResponse, error) {
```

Comment:
Maybe there should be a separate RPC (`HashRev`?), then; there might be cases where hashing the keys bucket with the current revision is desirable and there's no way to do that when reusing the current Hash RPC without having a flag like `quickHash`, which seems kind of hacky.